packages/react/src/components/Tooltip/TooltipHost.base.tsx

```diff
@@ -114,15 +112,13 @@ export class TooltipHostBase extends React.Component<ITooltipHostProps, ITooltip
             })}
             onMouseEnter={this._onTooltipMouseEnter}
             onMouseLeave={this._onTooltipMouseLeave}
-            {...getNativeProps(this.props, divProperties)}
+            {...getNativeProps(this.props, divProperties, ['id'])} // Make sure we use the id above
             {...tooltipProps}
           />
         )}
-        {isAriaPlaceholderRendered && (
-          <div id={tooltipId} role="none" style={hiddenContentStyle as React.CSSProperties}>
-            {content}
-          </div>
-        )}
+        <div id={tooltipId} role="none" style={hiddenContentStyle as React.CSSProperties}>
```

Comment:
```suggestion
        <div id={tooltipId} hidden style={hiddenContentStyle as React.CSSProperties}>
```

I'd suggest this (`hidden`, and no more need for `role=none`) to fix the double announcement thing. It worked for me with NVDA, but we can test more on Monday :).